ts/components/screens/BaseHeader.tsx

```diff
@@ -155,6 +158,7 @@ class BaseHeaderComponent extends React.PureComponent<Props, State> {
           numberOfLines={1}
           accessible={true}
           accessibilityRole={"header"}
+          style={{ color: titleColor ? IOColors[titleColor] : IOColors.black }}
```

Comment:
With this implementation, all the default title (non in DarkLayout) have changed from `bluegrey` to `black`.
In order to add this new parameter and keep the compatibility with the existing code, what do you think about:
```suggestion
style={titleColor ? { color: IOColors[titleColor] } : undefined}
```
In this way, without the `titleColor`, the component continues to have his previous behavior.

Btw I would add also a TODO, because it should be fixed in the future, with three parameters that act on the same component (`primary`, `dark` and `titleColor`) the expected behaviour could be very difficult to predict.

Reply:
By adding the default color I wanted to prevent the usage of the 3 different attributes in advantage of this new one, but just missed the default color was `bluegrey` instead of `black` 😅 

Maybe we can set the default as follows and add a TODO to remove the other references in an appropriate story?
```suggestion
          style={{ color: titleColor ? IOColors[titleColor] : IOColors.bluegrey }}
```

Reply:
Ok, in this case I would remove:
```typescript
white={isWhite}
```
from the `Text` props.
because is not used anymore (always overwritten by the style). What you think? In this case we could add a comment under `OwnProps` `primary` and `dark` to explain that are used only for the icons and not for the text?

Reply:
Yep good point! Added in 
fbba9a7